boards/others/esp32c3_supermini/Kconfig.defconfig

```diff
@@ -0,0 +1,11 @@
+# ESP32C3 supermini board configuration
+
+# Copyright (c) 2024 Arrel Neumiller
+# SPDX-License-Identifier: Apache-2.0
+if BOARD_ESP32C3_SUPERMINI
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
+	default 8192 # Heap size required with WiFi enabled
```

Comment:
Please remove the comment. That system heap is not used with the WIFi unless user specify it.